src/osemgrep/cli_ci/Project_metadata.ml

```diff
@@ -23,6 +23,62 @@ type 'e t = {
   extension : 'e;
 }
 
+let to_dict t =
+  let open JSON in
+  let or_null = Option.value ~default:Null in
+  Object
```

Comment:
It would be better to use ATD at some point to formally define the type for those interactions with the app.

Reply:
Maybe add a TODO note